CHANGELOG.md

```diff
@@ -6,9 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## 0.4.2-dev
 
+### Changed
+
+- Renamed `WINIT_HIDPI_FACTOR` environment variable to `WINIT_X11_SCALE_FACTOR`
+
 ### Fixed
 
 - Incorrect default config path in `--help` on Windows and macOS
+- Crash when starting on some X11 systems
 
```

Comment:
I kinda like how glutin links to winit changelogs, maybe that's a good idea here too. E.g. "Update winit dependency to 0.20.0-alpha4. See winit's CHANGELOG for more info."

Should we link to glutin in this changelog entry?

Reply:
>I kinda like how glutin links to winit changelogs, maybe that's a good idea here too. E.g. "Update winit dependency to 0.20.0-alpha4. See winit's CHANGELOG for more info."

glutin adds them to a changelog, because it makes sense for them. glutin just reexports a bunch of winit things, so update of winit in glutin is the same as updating glutin API to some point. The bump of libraries doesn't make sense to alacritty users and should live in a git history.

Reply:
I want to agree with you, but I'm wonding if it makes sense to show people things like:

> Breaking: Overhaul how Winit handles DPI: ...

Even though this should mostly be handled by our update, it may be useful to know. I'm fine just writing out our own entries for things we're aware are changed/fixed however.

Reply:
This is internal change, so it doesn't make sense to a user. User shouldn't even know about what winit/glutin is.

Reply:
Yeah, change log entries in winit/glutin should not matter to our users. The change log entries written in those libraries are meant for library consumers, so they focus on API/code changes.

Meanwhile our changelog is focused on actual users, so changes in behavior of the application itself are documented.

If there is a change in winit/glutin which is relevant to our users, we should just add it manually to our changelog. The 0.22.0 release is a great example for a change our users shouldn't care about at all. It should have no impact on our users.